examples/CMakeLists.txt

```diff
@@ -119,6 +119,16 @@ add_impactx_test(FODO.py
     examples/fodo/plot_fodo.py --save-png
 )
 
+# Python MADX: FODO Cell ######################################################
+#
+add_impactx_test(FODO_MADX.py
```

Comment:
similar to https://github.com/ECP-WarpX/impactx/blob/2d97982240de31a3c27a37f4049dd255ac00ea39/tests/python/CMakeLists.txt#L1-L2
```suggestion
# copy madx file
file(COPY ${ImpactX_SOURCE_DIR}/examples/fodo/input_fodo.madx DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/FODO_MADX.py)

add_impactx_test(FODO_MADX.py
```

We can also write this more generically above into the `add_impactx_test` function, e.g., copying all `*.madx` files found.